content/en/entities/ScheduledStatus.md

```diff
@@ -171,6 +173,13 @@ Returned from `GET /api/v1/scheduled_statuses`:
 **Version history:**\
 2.7.0 - added
 
+#### `params[application_id]` {#params-application_id}
+
+**Description:** ID of the Application that posted the status.\
```

Comment:
I think I would go a little further than a revert and change the description to say it's only kept for compatibility and should be ignored.